unittest/buildtool/bom_command_test.py

```diff
@@ -62,7 +62,7 @@ def make_default_options(options):
   options.build_number = 'OptionBuildNumber'
   options.bintray_org = 'test-bintray-org'
   options.bintray_debian_repository = 'test-debian-repo'
-  options.docker_registry = 'test-docker-registry'
+  options.artifact_registry = 'test-docker-registry'
```

Comment:
If this is really used as a docker registry, I'd keep the old variable name.  Or, if it's used for other artifacts too, I'm all for the new name, but then I'd change the value too.

Reply:
Indeed, `docker_registry` makes more sense. `container_registry` would be more precise perhaps (if changing).

It seems `artifact_registry` was introduced when container images were migrated to the new `spinnaker-community` GAR project. For a period images were published to old (`docker_registry`) and new (`artifact_registry`) GAR projects.

GAR also supports and we are now using it for `apt` packages.
`bintray*` doesn't make any sense anymore either.

Coming up in the TODO is sorting out `artifactSources` in the BOM in a manner that preferably works without a halyard change.
If we can't avoid it or it's too messy we could cut a new Halyard version that uses a newer BOM schema for and bump the required Halyard version for 1.27.0.

This PR's commit as it stands fixes the tests and makes sure the BOM generates correctly per the current use of a variable named `artifact_registry` for "container registry".